src/build.py

```diff
@@ -1373,7 +1373,7 @@ def AllBuilds():
       Build('fastcomp', Fastcomp),
       Build('emscripten', Emscripten),
       # Target libs
-      Build('wasi-sysroot', Wasi),
+      Build('wasi-sysroot', Wasi, os_filter=Filter(exclude=['windows'])),
```

Comment:
I think we also have to disable compiler-rt, libcxx and libcxxabi (since those get built into the was sysroot) as well as the 'bare' tests (since those test the wasi components)

Reply:
@dschuff thanks, I added exclusions for those, PTAL